business/tests/UnitTests.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+
+    <PackageId>UnitTests</PackageId>
+
+    <AssemblyName>UnitTests</AssemblyName>
+
+    <RootNamespace>UnitTests</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="coverlet.collector" Version="1.2.0" />
+    <PackageReference Include="Moq" Version="4.13.1" />
+    <PackageReference Include="NUnit" Version="3.12.0" />
```

Comment:
Funny, you're using NUnit, while I've just started a project with XUnit.

Reply:
Yes, I create this tests before your merge request. But i don't like a framework who call test 'Fact'...